spec/factories.rb

```diff
@@ -656,8 +656,10 @@
 
 FactoryBot.modify do
   factory :stock_location, class: Spree::StockLocation do
+    name { 'default' }
```

Comment:
This name doesn't make sense when using "OFN default" via `DefaultStockLocation.find_or_create`.